gr-zeromq/lib/base_impl.cc

```diff
@@ -72,6 +72,11 @@ base_sink_impl::base_sink_impl(int type,
 #endif
     }
 
+    // Set ZMQ_LINGER to 0 so socket won't infinitely block during teardown
+    // Re: Github #1132
+    const int linger_default = 1000;
```

Comment:
The comment says 0, but the value says 1000?

Reply:
Fixed the comments by removing the value.